Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -891,6 +891,9 @@ - (void)enableSystemPush
     if (block) {
         // If the app used [Leanplum setPushSetup:...], call the block.
         block();
+        if (response) {
```

Comment:
I am not sure what the appropriate behavior of this should be. Should this return `true`? `false`? Why?

Reply:
This callback short-circuits the registration process. When `block` exists, it is my assumption that the consumer would still configure the push requests via the dashboard and use `[Leanplum setPushSetup:]` to configure specific alert types. It's unlikely that both `block` and `response` would be a value.